eo2js/test/commands/link.test.js

```diff
@@ -39,4 +39,20 @@ describe('link', function() {
     assertFilesExist(link('--tests'), project, ['node_modules/mocha'])
     done()
   })
+  it('should not reinstall npm packages if package-lock.json exists', function(done) {
+    // First run to create initial files
+    link()
+    
+    // Get modification time of package-lock.json
```

Comment:
@HaidarJbeily7 inline comments and empty lines are not necessary

Reply:
Removed! @maxonfjvipon